pom.xml

```diff
@@ -36,7 +36,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>samplesvc-api</artifactId>
-      <version>10.49.7</version>
+      <version>10.49.9-SNAPSHOT</version>
```

Comment:
Did you mean to make this a snapshot version?

Reply:
Yes - changes to that module in another PR.  Need to leave it as SNAPSHOT until that change is merged, then will update this PR with release version (tedious, but the release only gets built when svcapi change is merged to master).